python/pyhail/context.py

```diff
@@ -10,7 +25,10 @@ class HailContext(object):
     :param SparkContext sc: The pyspark context.
     """
 
-    def __init__(self, sc):
+    def __init__(self, sc=None, log=None, quiet=False, append=False,
```

Comment:
`__init__` parameters are documented in the class doc string.  Steal the text from the Main Options object.